frontend/package.json

```diff
@@ -26,16 +26,19 @@
     "vue-i18n-extract-remove": "vue-i18n-extract --remove"
   },
   "dependencies": {
+    "@jellyfin/libass-wasm": "4.1.1",
     "@jellyfin/sdk": "0.0.0-unstable.202301101857",
     "@vueuse/components": "9.12.0",
     "@vueuse/core": "9.12.0",
+    "@vueuse/shared": "9.12.0",
```

Comment:
We don't need this, this is included in core

Reply:
Done